app/controllers/reservations_controller.rb

```diff
@@ -11,9 +11,9 @@ def show
 
     def create
         @screening = Screening.find(params[:screening_id])
-        #status should be somehow limited to confirmed, paied, cancelled
         @reservation = @screening.reservations.create(reservation_params)
         @reservation.ticket_desk_id = params[:ticket_desk_id]
```

Comment:
you can add it to reservation_params

Reply:
Is using validate inclusion  also a good idea for this?